resources/assets/components/WithReviewing/index.js

```diff
@@ -149,25 +151,21 @@ const reviewComponent = (Component, data) => {
       });
     }
 
-    // Update a signups quanity.
-    updateQuantity(signup, newQuantity) {
-      // Fields to send to /posts
-      const fields = {
-        northstar_id: signup.northstar_id,
-        campaign_id: signup.campaign_id,
-        campaign_run_id: signup.campaign_run_id,
-        quantity: newQuantity,
+    // Update a post's quantity.
+    updateQuantity(post, newQuantity) {
+      // Field to send to /api/v3/posts/:post_id
+      const field = {
+        quantity: parseInt(newQuantity),
       };
 
       // Make API request to Rogue to update the quantity on the backend
-      let request = this.api.post('posts', fields);
+      let request = this.api.patch('api/v3/posts/'.concat(post['id']), field);
```

Comment:
This might read a little better with a template string:

```js
let request = this.api.patch(`api/v3/posts/${post['id']}`, field);
```

Reply:
I'm not sure why this would have been passing before & not now, but we may want to add a named method to the `RogueClient` so we can [mock this call](https://github.com/DoSomething/rogue/blob/e64bfcedab841a5a1cef9a5733541e0c08f1d222/resources/assets/utilities/__mocks__/RogueClient.js) if it's run [in the test](https://github.com/DoSomething/rogue/blob/e64bfcedab841a5a1cef9a5733541e0c08f1d222/resources/assets/components/WithReviewing/test.js)?

Reply:
@DFurnes ah cool! I looked at that but wasn't sure if there was a way to add the OAuth authorization to that without affecting the old calls?

Reply:
Ahhh right, I forgot that's how we did this. It might make sense to make a `RogueOAuthClient` class that we could use for v3 endpoints & then we could configure and mock it separately?